core/templates/dev/head/pages/library-page/search-results/search-results.directive.html

```diff
@@ -14,16 +14,23 @@
       margin-right: 20px;
     }
   }
+  .d-lg-none-supplemental {
```

Comment:
Please use another more descriptive name for this.

Reply:
I have changed the name for the class.